projects/kit/components/text-area/text-area.template.html

```diff
@@ -45,12 +45,24 @@
         >
             <div class="input-wrapper">
                 <div class="relative">
-                    <div class="pseudo-content">{{value}}&nbsp;</div>
+                    <div
+                        class="pseudo-content"
+                        *tuiLet="pseudoContent as content"
+                        aria-hidden="true"
+                    >
+                        <span>{{content.fitted}}</span>
+                        <span
+                            class="pseudo-content__extra"
+                            *ngIf="content.extra"
```

Comment:
I believe there's no need for ngIf. It has no padding so empty span would already have 0 width and be invisible.

Reply:
remove `*ngIf`